src/components/include/cos_kernel_api.h

```diff
@@ -39,16 +40,16 @@ struct cos_compinfo {
 	vaddr_t vas_frontier, vasrange_frontier;
 	/* the source of memory */
 	struct cos_compinfo *memsrc; /* might be self-referential */
-	struct cos_meminfo mi;	     /* only populated for the component with real memory */
+	struct cos_meminfo local_mi;	     /* only populated for the component with real memory */
```

Comment:
See my previous comment.  I see no reason to broadly change all of this code for little reason.